nodes/src/nodes/tool_vectordb/IGlobal.py

```
@@ -0,0 +1,116 @@
+        """Dynamically import and instantiate the backend Store class."""
+        import importlib
+
+        module_path = _BACKEND_MODULES[backend]
+        mod = importlib.import_module(module_path)
+        store_cls = mod.Store
+        return store_cls(backend, conn_config, bag)
+
+    def validateConfig(self) -> None:
+        try:
+            cfg = Config.getNodeConfig(self.glb.logicalType, self.glb.connConfig)
+            backend = str((cfg.get('backend') or '')).strip().lower()
```

Ruff (0.15.7)

[warning] Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)